apps/charterafrica/src/components/OrganisationCard/OrganisationImageCard.js

```diff
@@ -0,0 +1,52 @@
+import { RichTypography } from "@commons-ui/core";
+import { Link, Figure } from "@commons-ui/next";
+import { Grid } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+const OrganisationImageCard = React.forwardRef(
+  function OrganisationImageCard(props, ref) {
+    const { avatarUrl: image, link, name } = props;
+
+    return (
+      <Link href={link.href} ref={ref} underline="none">
+        <Grid gap={1.2} container direction="column">
+          <Figure
```

Comment:
Whenever you use `Grid container`, it MUST be followed by `Grid item`s. If all you need is column direction, you can use `Stack` (or even a Box with proper attrs).